### PR TITLE
UX Improvements

### PR DESCRIPTION
**Pull request by Naatan**

### CODE REVIEW

File: internal/output/colorstyle/console_windows.go [hunk 1]
```diff
@@ -83,7 +94,7 @@ func (n *Styler) SetStyle(s Style, bright bool) {
 	if s == Default || s == Reset {
 		attr = bufferInfo.WAttributes
 	} else if s == Dim {
-		attr = bufferInfo.WAttributes
+		attr = attr & ^consoleColorMask | consoleStyleMap[Black]
```
Comment (mdrohmann) [inline internal/output/colorstyle/console_windows.go:97]: what is this doing?

**Review by mdrohmann — CHANGES_REQUESTED** [review]

File: internal/errs/errs.go [hunk 2]
```diff
@@ -94,11 +94,11 @@ func (e *ErrorWithTips) ErrorTips() []string {
 }
 
 func AddTips(err error, tips ...string) error {
-	if v, ok := err.(ErrorTips); ok {
-		v.AddTips(tips...)
-		return err
+	if _, ok := err.(ErrorTips); !ok {
+		err = newError(err, nil)
```
[the remark this answers is not in the document]
Comment (Naatan) [reply internal/errs/errs.go:98]: There isn't. Not every error necessarily implements ErrorTips already.

File: internal/prompt/cmd/test.go [hunk 3]
```diff
@@ -0,0 +1,28 @@
+package main
```
[the remark this answers is not in the document]
Comment (Naatan) [reply internal/prompt/cmd/test.go:1]: I want to keep it. Handy for testing things manually.

File: internal/output/colorstyle/console_windows.go (hunk 1 above)
Comment (Naatan) [reply internal/output/colorstyle/console_windows.go:97]: Showing bright black text, aka grey.

**Review by mdrohmann — APPROVED** [review]

**Pull request merged**